libraries/aws_ec2_instances.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'aws_backend'
+require 'aws_ec2_instance'
```

Comment:
what's the thinking behind adding this require?

Reply:
I was debugging an issue. I must have forgotten that. 
Thanks for catching that @rmoles.